tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain/README.md

```diff
@@ -0,0 +1,47 @@
+# Holocene Hardfork Upgrade - `SystemConfig`
+
+Status: READY TO SIGN
+
+## Objective
+
+Upgrades the `SystemConfig` contracts for the Holocene hardfork across multiple chains in the sepolia superchain.
+
+This upgrades the `SystemConfig` in the
+[v1.8.0-rc.4](https://github.com/ethereum-optimism/optimism/tree/v1.8.0-rc.4) release.
+
+## Pre-deployments
+
+- `SystemConfig` - `0x33b83E4C305c908B2Fc181dDa36e230213058d7d`
+
+## Simulation
+
+Please see the "Simulating and Verifying the Transaction" instructions in [NESTED.md](../../../NESTED.md).
+When simulating, ensure the logs say `Using script /your/path/to/superchain-ops/tasks/027-holocene-system-config-upgrade-and-init-multi-chain/NestedSignFromJson.s.sol`.
```

Comment:
```suggestion
When simulating, ensure the logs say `Using script /your/path/to/superchain-ops/tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain/NestedSignFromJson.s.sol`.
```
you fix this in the [mainnet pr](https://github.com/ethereum-optimism/superchain-ops/pull/454/files#diff-6e1818466a6f803978bd56a0a754e9b9de17381edc235a5f7f1a08f8f012a484)